doc/plutus-core-spec/plutus-core-specification.bib

```diff
@@ -704,3 +704,53 @@ @misc{BLS12-381
       howpublished = {\url{https://electriccoin.co/blog/new-snark-curve/}},
       url = {https://electriccoin.co/blog/new-snark-curve/}
 }
+
+@misc{FIPS-SHA2,
+  author = {Quynh Dang},
+  title = {{Secure Hash Standard (SHS)}},
+  year = {2012},
+  month = {March},
+  day={6},
+  publisher = {Federal Inf. Process. Stds. (NIST FIPS), National Institute of Standards and Technology, Gaithersburg, MD},
+  howpublished = {{FIPS PUB 180-4}},
+  doi = {https://doi.org/10.6028/NIST.FIPS.180-4},
+  url = {https://www.nist.gov/publications/secure-hash-standard},
+  language = {en},
+}
+
+@misc{FIPS-SHA3,
+  author = {Morris Dworkin},
+  title = {{SHA-3 Standard: Permutation-Based Hash and Extendable-Output Functions}},
+  year = {2015},
+  month = {August},
+  publisher = {Federal Inf. Process. Stds. (NIST FIPS), National Institute of Standards and Technology, Gaithersburg, MD},
+  howpublished = {FIPS PUB 202},
+  doi = {https://doi.org/10.6028/NIST.FIPS.202},
+  language = {en},
+}
+
+@misc{IETF-Blake2,
+    series =    {Request for Comments},
+    number =    7693,
+    howpublished =  {RFC 7693, \url{https://www.rfc-editor.org/info/rfc7693}},
+    publisher = {RFC Editor},
+    doi =       {10.17487/RFC7693},
+    url =       {https://www.rfc-editor.org/info/rfc7693},
```

Comment:
For IETF documents I usually cite `https://datatracker.ietf.org/`. Also seems that for other IETF documents we are citing datatracker, so maybe makes sense to be consistent here
https://datatracker.ietf.org/doc/html/rfc7693

Reply:
> For IETF documents I usually cite `https://datatracker.ietf.org/`. Also seems that for other IETF documents we are citing datatracker, so maybe makes sense to be consistent here https://datatracker.ietf.org/doc/html/rfc7693

I got that entry by clicking on the `bibtex` entry on the relevant datatracker page!  Oddly, datatracker gives you a reference to rfc-editor.org.  There's also an RFC which proposes a [bibtex format](https://datatracker.ietf.org/doc/html/draft-carpenter-rfc-citation-recs-01#section-6.2) for RFCs, but the datatracker bibtex doesn't conform to that. :(

We do refer to quite a number of RFCs though, so I'll try to make the bibtex consistent for those.